python/foglamp/services/core/api/browser.py

```diff
@@ -458,12 +458,7 @@ def where_clause(request, where):
         start = ts - length
         asset_code_list = asset_code.split(',')
         _readings = connect.get_readings_async()
-        for code in asset_code_list:
```

Comment:
We need to find a way for validation. Looks we are giving an empty set now if an invalid assetName or assetName/assetDatapoint or assetAttribute is supplied in request. Ideally it should be 404.